### PR TITLE
Add Create Menu options for all 3 LeapServiceProvider prefabs

### PR DESCRIPTION
**Pull request by MattGrayUL**
## Summary

Add Create Menu options for all 3 LeapServiceProvider prefabs to make it easier to add hand tracking to a scene

## Contributor Tasks

_These tasks are for the merge request creator to tick off when creating a merge request._

- [ ] Pair review with a member of the QA team.
- [ ] Add any release testing considerations to the MR for the next release.
- [x] Check any relevant CHANGELOG files have been updated.
- [x] Ensure documentation requirements are met e.g., public API is commented.
- [x] Consider any licensing/other legal implications for this MR e.g. notices required by any new libraries.
- [x] Add any relevant labels such as `breaking` to this MR.
- [ ] If this MR closes a Jira issue, make sure the fix version on the JIRA issue is set to the correct one.

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

[Use emojis in review threads to communicate intent and help contributors.](https://github.com/ultraleap/UnityPlugin/blob/develop/CONTRIBUTING.md#review-threads)

- [x] Code reviewed.
- [x] Non-code assets e.g. Unity assets/scenes reviewed.
- [x] Documentation has been reviewed. Includes checking documentation requirements are met and not missing e.g., public API is commented.
- [x] Checked and agree with release testing considerations added to MR for the next release.
- [ ] Make sure this works when the package in in /Packages/ as well as /Assets/ (via UPM and via .unitypackage)

## Closes JIRA Issue

Closes UNITY-781

### CODE REVIEW

File: Packages/Tracking/Core/Editor/Scripts/PrefabCreateMenu.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        public static void CreateServiceProviderXR()
+        {
+            CreatePrefab("Service Provider (XR)");
+        }
+
+        public static void CreatePrefab(string prefabName)
+        {
+            GameObject newObject = null;
+
+            if(newObject == null)
```
Comment (MaxPalmer-UH) [inline Packages/Tracking/Core/Editor/Scripts/PrefabCreateMenu.cs:39]: Why is this check needed?

Comment (MattGrayUL) [reply Packages/Tracking/Core/Editor/Scripts/PrefabCreateMenu.cs:39]: oops! this code iterated a lot and I left it in!

**Comment by MaxPalmer-UH** [comment]
Doesn't create the provider as a child of the selected object like other prefabs. Also, should the Ultraleap entry be in alphabetical order in the menu list?

**Comment by MattGrayUL** [comment]
> Doesn't create the provider as a child of the selected object like other prefabs. Also, should the Ultraleap entry be in alphabetical order in the menu list?

Just pushed these changes :)

**Comment by MaxPalmer-UH** [comment]
Child now works, but Ultraleap still appears at the top of the menu list.

**Comment by MattGrayUL** [comment]
I have placed Ultraleap at the bottom of the list (as best I can). the priority feature for MenuItems is not great for alphabetically ordering _within_ the existing root level list as it depends on whether any other packages add their own elements (you can only order by index)

**Comment by MaxPalmer-UH** [comment]
Great, that looks good now. Thanks for making the changes and sorry about the delay.

**Review by MaxPalmer-UH — APPROVED** [review]

**Pull request merged**